apps/aecore/lib/aecore/miner/worker.ex

```diff
@@ -306,87 +283,24 @@ defmodule Aecore.Miner.Worker do
 
   ## Internal
 
+  defp get_pool_values() do
+    pool_values = Map.values(Pool.get_pool())
+    max_txs_for_block = Application.get_env(:aecore, :tx_data)[:max_txs_per_block] - 1
+
+    if Enum.empty?(pool_values) or length(pool_values) < max_txs_for_block do
```

Comment:
I think the Enum.empty check isn't needed since length will just return 0 if it's empty. Also, the check should be length(pool_values) <= max_txs_for_block since right now it's checking if the length is < 99 meaning that there will be 99 transactions at most including the coinbase.